lab-isaac/src/component/dashboard-container/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {categoryCreate} from '../../action/category-actions';
+import CategoryForm from '../category-form';
+import CategoryItem from '../category-item';
+
+class DashboardContainer extends React.Component {
+  componentDidMount() {
+    this.props.categoryCreate({title: 'Pizza', budget: 100});
```

Comment:
always remove hard coded elements like this before submitting. It gets to be a lot when I come to the page and enter a category and there are already categories and then try to add an expense and cant really see where I should be adding it because there's already a lot on the page.